src/praisonai/praisonai/cli/commands/n8n.py

```
@@ -0,0 +1,321 @@
+        from praisonai.n8n import N8nClient
+        
+        client = N8nClient(base_url=n8n_url, api_key=api_key)
+        
+        if client.test_connection():
+            typer.echo(f"✅ Connected to n8n at {n8n_url}")
+            
+            # Try to list workflows to test API access
+            try:
+                workflows = client.list_workflows()
+                typer.echo(f"📊 Found {len(workflows)} workflows")
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)